integration-test/common/src/test/java/org/apache/uniffle/test/HealthCheckTest.java

```diff
@@ -1,38 +1,26 @@
-/*
```

Comment:
Why do we remove the license header?

Reply:
I moved this file, forgot to add it, let me add it